pages/DynamicPOC/index.tsx

```diff
@@ -160,16 +160,22 @@ const DynamicPOC: React.FC<{ questions: Question[] }> = ({ questions }) => {
     const y_inc = 8
 
     answers.forEach(function (item) {
-      doc.setFont('times', 'bold').text(item.question + '\n', x, y)
-      y += y_inc
+      const splitQuestion = doc.splitTextToSize(item.question, 200)
+      for (let i = 0; i < splitQuestion.length; i++) {
+        doc.setFont('times', 'bold').text(splitQuestion[i], x, y)
+        y += y_inc
+      }
       if (item.answer != null) {
         doc.setFont('times', 'normal').text('\t' + item.answer + '\n\n', x, y)
         y += y_inc
       }
       if (item.formAnswer.type === QuestionType.TEXT) {
-        doc
-          .setFont('times', 'normal')
-          .text('\t' + item.formAnswer.userAnswer + '\n\n', x, y)
+        const splitAnswer = doc.splitTextToSize(item.formAnswer.userAnswer, 200)
```

Comment:
Is this a function to help write the formanswers into a pdf?

Reply:
yup, it's a method in jspdf to split the text so it doesn't go off the page